modules/library/data/src/test/java/org/geotools/data/memory/MemoryDataStoreTest.java

```diff
@@ -911,11 +907,11 @@ public void testGetFeatureSourceRoad() throws IOException {
         assertEquals(rd12Bounds, some.getBounds());
         assertEquals(some.getSchema(), road.getSchema());
 
-        DefaultQuery query = new DefaultQuery("road", rd12Filter, new String[]{"name",});
+        DefaultQuery query = new DefaultQuery("road", rd12Filter, new String[]{"name", "geom"});
 
```

Comment:
I have no clue how this test managed to function before this change - without geom in the featureType we shouldn't expect to get geometries back, should we?